app/helpers/application_helper.rb

```diff
@@ -494,8 +494,7 @@ def user_set?(field)
   end
 
   def notifications
-    content_tag :div, id: 'toast-notifications-container',
-                      'data-notifications': toast_notifiations_data.to_json.html_safe do
```

Comment:
how are you planning to pass the data? using API? not sure i understand the reasoning for this change?

Reply:
Accidentally removed the changed line after this, thanks for noticing :)